src/content/en/2020/compression.md

```diff
@@ -4,20 +4,272 @@ part_number: IV
+veluca93_bio: Luca Versari is a software engineer at Google, working on <a href="https://gitlab.com/wg1/jpeg-xl">JPEG XL</a>. He's finishing a PhD on graph compression and has a background in mathematics.
 discuss: 2055
 results: https://docs.google.com/spreadsheets/d/1NKbP4AqMkgCNCsVD3yLhO2d0aqIsgZ7AGLEtUDHl9yY/
 queries: 19_Compression
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
+featured_quote: Using HTTP compression makes a website load faster and therefore guarantees a better user experience.
+featured_stat_1: 23%
+featured_stat_label_1: Compressed requests which use brotli
+featured_stat_2: 77%
+featured_stat_label_2: Compressed requests which use gzip
+featured_stat_3: 74%
+featured_stat_label_3: Websites that pass the Lighthouse audit with maximum score on text compression
 unedited: true
 ---
+
+## Introduction
+
+
+Using HTTP compression makes a website load faster and therefore guarantees a better user experience. Running no compression on HTTP makes for a worse user experience, may affect the growth rate of the related web service and affects search rankings. Using compression likely produces a web experience that performs better on metrics such as faster Largest Contentful Paint. Using compression reduces [page weight](./page-weight), improves [web performance](./performance), and therefore is an important part of [search engine optimization](./seo).
+
+
+While lossy compression is often acceptable for images and other [media](./media) types, for text we want to use lossless compression, i.e. recover the exact text after decompression.
+
+## What type of content should we compress?
+
+For most text-based assets, such as [HTML](./markup), [CSS](./css), [JavaScript](./javascript), [JSON](https://www.json.org) or SVG, as well as certain non-text formats such as woff, ttf, ico, using compression is recommended.
+Here is an overview over what compression methods are currently used for different content types:
+
+{{ figure_markup(
+  image="compession-methods-by-content-type.png",
+  caption="Compression Methods for Different Content Types",
+  description="This breaks down what compression methods, if any, are used for all the content types which are not images.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxUj8-0vKTqPAblIXqekSbiRh1D1lEuA3gVD9w23qwGPtJRE8FbgrURfPAgfFZX2l0t84Wy5ZAGqzR/pubchart?oid=1658254159&format=interactive",
+  sheets_gid="107138856",
+  sql_file="19_01.type_of_content_encoding.sql"
+  )
+}}
+The figure shows the percentages of the request of a certain content type using either brotli, gzip or no text compression.
+It is surprising that while all those content types would profit from compression, the range of percentages varies widely over the different content types:  only 44% use compression for `text/html` against  93% `application/x-javascript`.
```

Comment:
I realize that I am a bit late to the party, but it's not _that_ surprising: the bulk of JavaScript are static files served from CDNs, while a good proportion of HTML is generated dynamically.

Reply:
I still think it's surprising and fine to leave as is. CDNs aren't use that much (not like it's 93% of the time) and dynamic compression is still relatively cheap compared to network costs, so personally I still think it should be enabled even for dynamic content.